### ISSUE TITLE
Mumble takes long time to start

### ISSUE THREAD

**Issue by vimpostor** [mumble-voip/mumble#5157]
For every Audio backend being initialized, Mumble waits a whole second. This ends up in Mumble needing 3 seconds to start:

```
<D>2021-07-01 22:22:17.140 PortAudio V19.7.0-devel, revision unknown from libportaudio.so
<X>2021-07-01 22:22:18.141 PipeWire 0.3.31 from libpipewire-0.3.so
<D>2021-07-01 22:22:19.141 JACK 1.9.18 from libjack.so
```
```c++
void PipeWireInit::initialize() {
	pws = std::make_unique< PipeWireSystem >();

	// Wait for initialization process to complete.
	pws->m_lock.lock();
	pws->m_waiter.wait(&pws->m_lock, 1000);
	pws->m_lock.unlock();
}
```

Mumble 1.3 didn't need so long to start. Is the 1000 ms wait really needed?

System information: Latest Mumble on Arch Linux with Pipewire 0.3.31 as pulseaudio and jack bridge

**Comment by vimpostor** [comment]
Actually I just noticed Mumble 1.3 also waits 1000ms per audio backend being loaded. What even is the purpose of that mutex waiting? Surely instead of waiting for an arbitrary amount of time one could get an interrupt when that ominous "initialization process" is done.

What initialization process is even referred to in the above comment?

**Comment by davidebeatrici** [comment]
The timer is indeed not ideal and should be removed.

There are some backends that are initialized in multiple steps (which could fail):
https://github.com/mumble-voip/mumble/blob/1bcbb79706a3aa8f7127469f0347bebd6d812c32/src/mumble/PAAudio.cpp#L125-L173

I think the original's purpose of the code (present for all backends) was to wait until a separate initialization thread finished.
Such thread does not exist though, as far as I can see.

**Comment by Krzmbrzl** [comment]
The additional wait time in 1.4.x comes from more backends being enabled by default. Thus more things are loaded and that means more of these timers run.

@davidebeatrici so we wait on am async operation that doesn't actually exist because the operation runs in the same thread? 👀

**Comment by davidebeatrici** [comment]
Exactly.

**Comment by vimpostor** [comment]
I think we should then just remove that code, shouldn't we?

BTW I also just noticed I might have opened a duplicate of #5022

**Comment by davidebeatrici** [comment]
Let's remove it for all backends.

I think it's the cause of #5022 as well.
